example/android/gradle.properties

```diff
@@ -36,7 +36,7 @@ android.enableJetifier=true
 
 # Enable new architecture, i.e. Fabric + TurboModule - implies USE_FABRIC=1.
 # Note that this is incompatible with web debugging.
-#newArchEnabled=true
+newArchEnabled=true
```

Comment:
Can you also add `RCT_NEW_ARCH_ENABLED=1` to `pod install`?

https://github.com/react-native-async-storage/async-storage/blob/f4e5d92141206d7f2d8037113afcc7130bfb0ec6/.github/workflows/ci.yml#L75

https://github.com/react-native-async-storage/async-storage/blob/f4e5d92141206d7f2d8037113afcc7130bfb0ec6/.github/workflows/ci.yml#L112

Reply:
Done in https://github.com/react-native-async-storage/async-storage/pull/910/commits/2e939383f30c9878ad1b23831e666c9663452608